src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -14,6 +14,10 @@ namespace Microsoft.CodeAnalysis.FindSymbols
 {
     public static partial class SymbolFinder
     {
+        /// <summary>
+        /// Obsolete.  Use <see cref="FindSymbolAtPositionAsync(SemanticModel, int, Workspace, CancellationToken)"/>.
+        /// </summary>
+        [Obsolete("Use FindSymbolAtPositionAsync instead.")]
```

Comment:
Can we at least have an internal comment of why we obsoleted it? I'm guessing because it's doing bad blocking?